### PR TITLE
Tab focus cycling in Modal

### PR DESCRIPTION
**Pull request by plbabin**
This updates move the focus to the first or last in the list based on if the user reach an edge node inside a Modal

Screen Recording 2020-04-20 at 05 02 PM

_**edit**_

### Modal icon size
We also update the `Modal` component to add a new prop  `iconSize` to modify the size of the header icon

### VerificationCode autoSubmit
We split the autoSubmit prop into two different prop (`autoSubmitPaste` `autoSubmitKeyUp`), to allow for more customization. We can now trigger the auto submit on paste or keyup only, or for both.

### CODE REVIEW

File: src/components/Modal/Modal.jsx [hunk 1]
```diff
@@ -199,7 +199,8 @@ class Modal extends React.PureComponent {
 
     if (focusedNodeIndex === 0) {
       event.preventDefault()
-      focusableNodes[focusableNodes.length - 1].focus()
+      const i = focusableNodes.length - 1
```
Comment (cocoabythefire) [inline src/components/Modal/Modal.jsx:202]: should we defend for the possibility that focusableNodes.length could be 0 which make this try to access index of -1?

File: src/components/Modal/Modal.jsx [hunk 2]
```diff
@@ -64,6 +64,7 @@ class Modal extends React.PureComponent {
     isHsApp: PropTypes.bool,
     isOpen: PropTypes.bool,
     icon: PropTypes.string,
+    iconSize: PropTypes.string,
```
Comment (cocoabythefire) [inline src/components/Modal/Modal.jsx:67]: can we accept string or number and be able to handle either? just a nice to have

File: src/components/Modal/Modal.jsx (hunk 1 above)
Comment (plbabin) [reply src/components/Modal/Modal.jsx:202]: yes good catch, I did fixed it in 2d6cabbf

File: src/components/Modal/Modal.jsx (hunk 2 above)
Comment (plbabin) [reply src/components/Modal/Modal.jsx]: 👍 all done in 2d6cabbf

**Pull request merged**